main.tf

```diff
@@ -18,7 +18,7 @@ module "logs" {
   stage                    = "${var.stage}"
   name                     = "${var.name}"
   delimiter                = "${var.delimiter}"
-  attributes               = ["logs"]
+  attributes               = ["${compact(concat(var.attributes, list("origin", "logs")))}"]
```

Comment:
Do we need to worry about de-duping `origin` `logs`? Although I dont see a reason for the caller to provide at this point

Reply:
I'd rather not as then there's a risk that names aren't truly unique